sickbeard/tv.py

```diff
@@ -1020,7 +1020,7 @@ def _load_from_db(self):
         sql_results = main_db_con.select(b'SELECT * FROM imdb_info WHERE indexer_id = ?', [self.indexerid])
 
         if not sql_results:
-            logger.log(u'{0}: Unable to find IMDb show info in the database'.format(self.indexerid))
+            logger.log(u'Unable to find the following IMDb show info in the database: {0} '.format(self.indexerid))
```

Comment:
you are doing exactly the oposite from my PR

it should be:
logger.log(u'{id}: Unable to find the following IMDb show info in the database: {show}.format(id=self.indexerid,show=self.name))